startup/GafferUSD/usdInstancers.py

```diff
@@ -0,0 +1,150 @@
+##########################################################################
+#
+#  Copyright (c) 2025, Image Engine Design Inc. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#      * Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials provided with
+#        the distribution.
+#
+#      * Neither the name of John Haddon nor the names of
+#        any other contributors to this software may be used to endorse or
+#        promote products derived from this software without specific prior
+#        written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import IECore
+
+import Gaffer
+import GafferScene
+
+# Register a render adaptor that converts PointsPrimitives in the usd:pointInstancers set
+# into actual Instancers. This supports how USD PointInstancers are currently loaded from
+# USD files. In the future, they may be first class objects, and this will be unnecessary.
+
+def __usdPointInstancerAdaptor() :
```

Comment:
I think we'd benefit from some unit tests for this node. I suggest we move it to `GafferUSD._PointInstancerAdaptor` (note the underscore for protection), add tests in `GafferUSDTest.PointInstancerAdaptorTest` and then keep the registration here as a one-liner.

Reply:
Should it be stored as a .gfr as well, for consistency with the PromoteInstances node?

Reply:
You didn't feel strongly about this ... I ended up deciding not to store it as a .gfr, because the rewiring of an Instancer necessary to make it recursive wouldn't be very clear that way.